libkbfs/block_journal.go

```diff
@@ -299,10 +306,20 @@ func (j *blockJournal) appendJournalEntry(
 	return ordinal, nil
 }
 
-func (j *blockJournal) length() (uint64, error) {
+func (j *blockJournal) length() uint64 {
```

Comment:
Orthogonal to this PR, but why does this have a pointer receiver while the `.length()` functions of other journal structs do not?